transforms/universal/pii_redactor/python/README.md

```diff
@@ -0,0 +1,65 @@
+# PII Redactor Transform
+
+This transform redacts Personally Identifiable Information (PII) from the input data.
+
+The transform leverages the [Microsoft Presidio SDK](https://microsoft.github.io/presidio/) for PII detection and uses the Flair recognizer for entity recognition.
+
+### Supported Entities
+
+The transform detects the following PII entities by default:
+- **PERSON**: Names of individuals
+- **EMAIL_ADDRESS**: Email addresses
+- **ORGANIZATION**: Names of organizations
+- **DATE_TIME**: Dates and times
+
+You can configure the entities to detect by passing the required entities as argument param ( **--pii_redactor_entities** ).
+To know more about different entity types supported - [Entities](https://microsoft.github.io/presidio/supported_entities/)
+
+### Redaction Techniques
+
+Two redaction techniques are supported:
+- **replace**: Replaces detected PII with a placeholder (default)
+- **redact**: Removes the detected PII from the text
+
+You can choose the redaction technique by passing it as an argument parameter (**--pii_redactor_operator**).
+
+## Input and Output
+
+### Input
+
+The input data should be a `py.Table` with a column containing the text where PII detection and redaction will be applied. By default, this column is named `contents`, but this can be configured.
+
+**Example Input Table Structure:**
+
+| contents               | doc_id |
+|------------------------|--------|
+| My name is JohnDoe    | doc001 |
+| I work at apple        | doc002 |
+
+### Output
+
+The output table will include the original columns plus an additional column `new_contents` with redacted text.
+
+**Example Output Table Structure:**
+
+| contents               | doc_id | new_contents           |
+|------------------------|--------|------------------------|
+| My name is JohnDoe    | doc001 | My name is <NAME>     |
+| I work at apple        | doc002 | I work at <ORGANIZATION> |
```

Comment:
Do you have an example for "anonamize"?

Reply:
I assume the question is what happens when the operator is given as redact @daw3rd . If yes, yeah I can provide one sample case in this READme. Else can you elaborate on this one?

Reply:
Yes, a "redact" example would be good (and associated test of both options).  

I think I am a bit confused by the usage of the term "anonymize" in the code/readme.  To me this word might mean to replace, for example, a credit card number of a fake value like 01234567890.  That is, the concept is left intact in the document, but its value is not specific to any PII type.  The fix may be simply to remove the word "anonymize" from the transform.

Reply:
removed that in transform file and updated the READme